### PR TITLE
fix: Turn check that activity is successful back on

### PR DESCRIPTION
**Pull request by abayer**
Signed-off-by: Andrew Bayer <andrew.bayer@gmail.com>

### CODE REVIEW

File: test/helpers/test.go [hunk 1]
```diff
@@ -363,16 +365,10 @@ func (t *TestOptions) ThereShouldBeAJobThatCompletesSuccessfully(jobName string,
...
 
-	By(fmt.Sprintf("checking that the activity %s has succeeded", activityKey), func() {
-		// TODO lets temporarily disable this assertion as we have an issue on our production cluster with build statuses not being set correctly
-		// TODO lets put this back ASAP once we're on tekton!
-		/*
-			Expect(activity.Spec.Status.IsTerminated()).To(BeTrue())
-			Expect(activity.Spec.Status.String()).Should(Equal("Succeeded"))
-		*/
+			// TODO: Fix the regex in get_activities_parser to not treat "Succeeded Version: 0.0.1" as the status. I'm too lazy right now.
+			Expect(activity.Status).Should(HavePrefix("Succeeded"))
```
Comment (hferentschik) [inline test/helpers/test.go]: So is this good to merge?

Comment (abayer) [reply test/helpers/test.go]: Yes, it is.

**Pull request merged**